### DISCUSSION TITLE
Social icons

### DISCUSSION THREAD

**Discussion by csulit** [tailwindlabs/heroicons#588] [Ideas]
Please add social icons similar to this.

![image](https://user-images.githubusercontent.com/31933946/152293865-e2f99755-c772-4981-b162-3b2ba8e4e1a7.png)

**Comment by cgarvis** [comment]
Add your votes to https://github.com/tailwindlabs/heroicons/discussions/571

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
